src/main/java/io/kestra/plugin/microsoft365/outlook/Get.java

```diff
@@ -0,0 +1,303 @@
+        )
+        private final String bodyContent;
+
+        @Schema(
+            title = "Body type",
+            description = "Content type of the email body (HTML or TEXT)"
+        )
+        private final String bodyType;
+
+        @Schema(
+            title = "Body preview",
+            description = "Preview text of the message body"
+        )
+        private final String bodyPreview;
+
+        @Schema(
+            title = "Sender",
+            description = "Email address of the sender"
+        )
+        private final String sender;
+
+        @Schema(
+            title = "From",
+            description = "Email address in the from field"
+        )
+        private final String from;
+
+        @Schema(
+            title = "To recipients",
+            description = "List of email addresses in the to field"
+        )
+        private final List<Recipient> toRecipients;
+
+        @Schema(
+            title = "CC recipients",
+            description = "List of email addresses in the cc field"
+        )
+        private final List<Recipient> ccRecipients;
+
+        @Schema(
+            title = "BCC recipients",
+            description = "List of email addresses in the bcc field"
+        )
+        private final List<Recipient> bccRecipients;
+
+        @Schema(
+            title = "Received date/time",
+            description = "Date and time the message was received"
+        )
+        private final OffsetDateTime receivedDateTime;
+
+        @Schema(
+            title = "Sent date/time",
+            description = "Date and time the message was sent"
+        )
+        private final OffsetDateTime sentDateTime;
+
+        @Schema(
+            title = "Is read",
+            description = "Whether the message has been read"
+        )
+        private final Boolean isRead;
+
+        @Schema(
+            title = "Has attachments",
+            description = "Whether the message has attachments"
+        )
+        private final Boolean hasAttachments;
+
+        @Schema(
+            title = "Importance",
+            description = "Importance level of the message"
+        )
+        private final String importance;
+
+        @Schema(
+            title = "Conversation ID",
+            description = "Identifier of the conversation thread"
+        )
+        private final String conversationId;
+
+        @Schema(
+            title = "Conversation index",
+            description = "Index of the message in the conversation"
+        )
+        private final String conversationIndex;
+
+        @Schema(
+            title = "Internet message ID",
+            description = "Internet message ID of the email"
+        )
+        private final String internetMessageId;
+
+        @Schema(
+            title = "Attachments",
+            description = "List of attachment information"
+        )
+        private final List<AttachmentInfo> attachments;
```

Comment:
Not sure we want to expose as outputs classes from the `com.microsoft.graph.models` package directly here.

Could you add a `io.kestra.plugin.microsoft365.outlook.domain` package with your business data classes to map data coming from Outlook and use them into the outputs?

Plus, maybe you could refactor some outputs when they give the same data.